packages/aragon-cli/config/truffle.default.js

```diff
@@ -10,7 +10,10 @@ const HDWalletProviderPrivkey = require('truffle-hdwallet-provider-privkey')
 const DEFAULT_MNEMONIC =
   'explain tackle mirror kit van hammer degree position ginger unfair soup bonus'
 
-const defaultRPC = network => `https://${network}.infura.io`
+const ARAGONCLI_PROJECT_ID = 'd146fdf20d88476d832edd5d65fc2339'
+
+const defaultRPC = network =>
+  `https://${network}.infura.io/v3/${ARAGONCLI_PROJECT_ID}`
```

Comment:
Hmm, don't know about this; is `mainnet.eth.aragon.network` too slow?

Reply:
It was cause we had a [`Legacy access request rate exceeded` issue](https://community.infura.io/t/legacy-access-request-rate-exceeded/490) when doing something like: `aragon apm packages open.aragonpm.eth`. I think cause there are too many repos on that one.

Reply:
I create an Infura Project id with my account, but maybe Aragon already have one?

Reply:
We do have one, but we shouldn't embed it into the code like this (because people may find it and abuse it).

Why not just use `https://mainnet.eth.aragon.network`?

Reply:
Oh cool, that works as well. This was a legacy value I was reusing, but definitely we should use Aragon's rpc 👍